### DISCUSSION TITLE
Leistung/Netz im Display anzeigen

### DISCUSSION THREAD

**Discussion by thinklerde** [hoylabs/OpenDTU-OnBattery#632] [Ideas]
Gibt es die Möglichkeit die Leistung/Netz auch im Display anzeigen zu lassen? Gibt es das schon und ich hab es bisher noch nicht gefunden? Ist da was geplant? Oder unerwünscht?

Danke im Voraus!!

**Comment by spcqike** [comment]
ich verweis mal auf die gleiche Frage von letzter Woche

https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/620

**Reply by schlimmchen** [reply]
Siehe #658
